### PR TITLE
gh-463: NHS branding applied. Also updated Requests page so that "Req…

### LINKED ISSUE #463: Apply NHS branding
Task to make the required changes to the sde-resources to make sure that the web applications are branded correctly.

### PR DESCRIPTION
**Pull request by NigelPalmer**
…uests to Authorise" list is shown if the user is an approver of projects or deparments.

### CODE REVIEW

File: src/app/secure-data-environment/components/sde-requests/sde-requests.component.ts [hunk 1]
```diff
@@ -17,17 +17,38 @@ limitations under the License.
 SPDX-License-Identifier: Apache-2.0
 */
 import { Component, OnInit } from '@angular/core';
-import { RequestsListMode } from '@maurodatamapper/sde-resources';
+import {
+  RequestsListMode,
+  DepartmentsService,
```
Comment (giovannaOCC) [inline src/app/secure-data-environment/components/sde-requests/sde-requests.component.ts]: DepartmentsService doesn't seems to be used within the file.
was it meant to be used and something went forgotten?

**Comment by giovannaOCC** [comment]
When you have no bookmarks saved, by clicking on the Bookmarks menu on the top-right, you will get a message (alert) saying that you don't have any. 
Message is correct but the background is light blue -> it comes from  " `.alert-info`  in `_alert.scss` file (directly from bootstrap)

**Review by giovannaOCC — APPROVED** [review]
This PR is approved as considered "stable/good" enough and worth to be 
integrated for the DEMO due tomorrow

There are a couple of minor things to be fixed.
A follow-up issue will be created covering all of them.

File: src/app/secure-data-environment/components/sde-requests/sde-requests.component.ts (hunk 1 above)
Comment (NigelPalmer) [reply src/app/secure-data-environment/components/sde-requests/sde-requests.component.ts]: Departments service reference was removed within the commit for test fixes.

**Comment by NigelPalmer** [comment]
Follow-up task is here: #467

**Pull request merged**